content/changelog/2025-11-07.md

```diff
@@ -0,0 +1,125 @@
+
+This change applies automatically to all Free plan users—no action required. For more information about plan limits, see [Neon plans](/docs/introduction/plans).
+
+## Backup schedule API (Beta)
+
+You can now manage automated snapshot schedules via the Neon API. Previously, backup schedules could only be configured through the Console. With the new API endpoints, you can programmatically view and update backup schedules for your branches, enabling infrastructure-as-code workflows and automated backup management.
+
+**Available endpoints:**
+
+- `GET /projects/{project_id}/branches/{branch_id}/backup_schedule` — View the current backup schedule for a branch
+- `PUT /projects/{project_id}/branches/{branch_id}/backup_schedule` — Update the backup schedule configuration
+
+This makes it easier to standardize backup policies across projects and integrate snapshot scheduling into your deployment pipelines.
+
+For more information, see [Backup & restore](/docs/guides/backup-restore).
+
+## Vercel integration now supports current Neon pricing plans
+
+Vercel integration users now have full access to Neon's current usage-based pricing plans (Free, Launch, and Scale). Previously, Vercel integration users were limited to Neon's legacy plans. Free plan Vercel users have been automatically migrated to the new Neon Free plan. This change gives you access to all the latest features and pricing options, including:
+
+- Usage-based pricing with $5/month minimum on paid plans
+- Enhanced compute options and autoscaling
+- Advanced features like database branching for preview deployments
+- Flexible project and branch limits
+
+To upgrade your plan, visit the **Storage** tab in your Vercel Dashboard, select your database, and navigate to **Settings** > **Change Configuration**.
+
+For more information, see [Neon plans](/docs/introduction/plans).
+
+<details>
+<summary>**Fixes & improvements**</summary>
+
+- **Backup & restore**
+  - Fixed broken root branch selector in Point-in-time restore UI
+  - Fixed confusing snapshot sort order: snapshots now consistently show most recent first
+  - Added a helpful link from the Backup & Restore page to configure your PITR settings
+  - Changed default retention period for daily snapshot schedules from 1 to 14 days
+- **Console**
+  - Current project now always appears in the project picker, even when you have many projects
+  - Improved real-time state updates for branches and endpoints
+- **Vercel integration**
+  - Automatic cleanup of deployment records when branches are deleted
+- **Neon Auth**
+  - Fixed Neon Auth cleanup when deleting branches
+- **Database roles**
+  - The `authenticated` and `anonymous` roles are now granted to `neondb_owner`, enabling automated testing of grants and RLS policies using pgTAP
+- **Documentation**
+  - Updated documentation to reflect new Vercel integration plan availability
+  - Recalculated pricing examples in documentation to reflect new compute rates
```

Comment:
This is a partial description, compute prices were also updated. Might not be worth mentioning. We cover it in the compute price update above. If we add branch new docs, like a guide, them maybe mention it? There have been more PRs merged that this in the website repo, which are not captured.
- CLI init docs
- Drizzle docs update
- New Phenix guide
Some others

Not sure any of these need to be mentioned ..